tests/extensions/md/test_renderer.py

```diff
@@ -48,38 +50,25 @@ def test_render_md_cache_result(self, renderer):
         assert renderer.cache_result is True
 
     def test_render_md(self, test_md_file_path, assets_url, export_url):
-        metadata = ProviderMetadata('test', '.md', 'text/plain', '1234', 'http://wb.osf.io/file/test.md?token=1234')
-        renderer = MdRenderer(metadata, test_md_file_path, url, assets_url, export_url)
-        body = renderer.render()
-        inbody = """
-<h1>Heading</h1>
-<h2>Sub-heading</h2>
-<h3>Another deeper heading</h3>
-<p>Paragraphs are separated
-by a blank line.</p>
-<p>Leave 2 spaces at the end of a line to do a<br />
-line break</p>
-<p>Text attributes <em>italic</em>, <strong>bold</strong>, 
-<code>monospace</code>.</p>
-<p>A <a href="http://example.com">link</a>.
-[28]</p>
-<p>Shopping list:</p>
-<ul>
-<li>apples</li>
-<li>oranges</li>
-<li>pears</li>
-</ul>
-<p>Numbered list:</p>
-<ol>
-<li>apples</li>
-<li>oranges</li>
-<li>pears</li>
-</ol>
-<p>The rain---not the reign---in
-Spain.</p>
-<p>&lt;script&gt;
-alert("Hello world");
-&lt;/script&gt;</p>
-"""
-        assert inbody in body
 
+        metadata = ProviderMetadata(
+            'test',
+            '.md',
+            'text/plain',
+            '1234',
+            'http://wb.osf.io/file/test.md?token=1234'
+        )
+
+        md_renderer = MdRenderer(metadata, test_md_file_path, url, assets_url, export_url)
+        body = md_renderer.render()
+        in_body = 'The rain---not the reign---in\nSpain.\n\n&lt;script&gt;\nalert("Hello world");\n&lt;/script&gt;'
+        assert in_body in body
+
+    # TODO: it is hard to test since rendering is performed by front-end javascript, if possible test the following:
```

Comment:
It is hard to test front-end based rending for `.md` and there a lot to test. I will try to add more tests before/during phase 2 review if we think it is necessary.

Reply:
Not sure how relevant this is, but I mostly just tried to test bleach quickly here. It may be worth while to make sure proper tags are being bleached, and javascript would get escaped out etc. The above is already testing for that a little.
Might be worth a parameterized test to test all the `BLEACH_WHITELIST` tags or whatever that setting got called.